modelreg/templates/registration/activation_email.txt

```diff
@@ -1,18 +1,16 @@
+Hi {{user}}
```

Comment:
The whole email should be wrapped in a {%blocktrans%}.

Note that to inject variables, you can do this: `{%blocktrans with domain=site.doman, url=... %} and so on.